--- a/krita/main.cc
+++ b/krita/main.cc
@@ -716,9 +716,9 @@
     /// Initialize application info, it will be used by both, Qt and
     /// DrKonqi of the host system
     KAboutData aboutData("krita",
-                            "Krita",
+                            QStringLiteral("Krita Baron Edition"),
                             KritaVersionWrapper::versionString(true),
-                            "", // TODO: "short description" needs new string exception
+                            i18n("Unofficial modified build of Krita with Baron integration"),
                             KAboutLicense::GPL,
                             i18nc("@info:credit", "© 1999–2026 The Krita Developers"));
     aboutData.setHomepage(QStringLiteral("https://krita.org"));
